src/renderer/component/channelTile/view.jsx

```diff
@@ -36,27 +38,54 @@ class ChannelTile extends React.PureComponent<Props> {
   }
 
   render() {
-    const { claim, navigate, isResolvingUri, totalItems, uri } = this.props;
-    let channelName, channelId;
+    const { claim, navigate, isResolvingUri, totalItems, uri, size } = this.props;
```

Comment:
Should probably split this into multiple lines

Reply:
prettier will split it if it exceeds x characters